index.css

```diff
@@ -0,0 +1,617 @@
+  color: #3e3e3e;
+}
+
+.program-bg {
+  background-repeat: repeat;
+  background-color: rgb(39, 42, 50);
+  background-image: url('/assets/bg-sponsors.jpg');
+}
+
+.program-section {
+  /* display: grid; */
+  grid-template-columns: 15% 30% 55%;
+  align-items: center;
+  justify-content: space-between;
+  background-color: rgba(255, 255, 255, 0.15);
+  transition: all 0.3s ease-in-out;
+}
+
+.program-section:hover {
+  border: 1px solid #fff;
+  cursor: pointer;
+}
+
+.grid {
+  display: grid;
+  grid-template-columns: 1fr;
+  padding: 0 20px;
+}
+
+/* .flex-program {
+  display: grid;
+  grid-template-columns: 15% 35% 50%;
+  align-items: center;
+  justify-content: space-between;
+} */
+
+.text-card-1 {
+  font-family: cocogoose, sans-serif;
+  font-size: 12px;
+  font-weight: normal;
+  line-height: 16px;
+  text-align: left;
+}
+
+.text-card-2 {
+  font-family: cocogoose, sans-serif;
+  font-size: 9px;
+  font-weight: normal;
+  line-height: 16px;
+  text-align: left;
+}
+
+.p-top {
+  padding-top: 2rem !important;
+  background: rgb(220, 53, 69);
+}
+
+.img-1 {
+  max-width: 40px;
+  max-height: 100%;
+}
+
+.img-2 {
+  width: 100%;
+  height: 100%;
+}
+
+.img-3 {
+  width: 100px;
+  height: auto;
+}
+
+.guide-line {
+  width: 4rem;
+  height: 2px;
+  background-color: #be4d5a;
+}
+
+.grid-2 {
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: 40px;
+  padding: 0 20px;
+}
+
+.flex-speaker {
+  display: block;
+}
+
+.speaker-list {
+  overflow: auto;
+  height: auto;
+}
+
+.speaker-img {
+  float: left;
+  margin-right: 30px;
+}
+
+.speaker {
+  width: 100%;
+  height: auto;
+  margin-bottom: 1.5rem;
+  margin-left: 0;
+}
+
+.name {
+  font-family: cocogoose, sans-serif;
+  font-size: 12px;
+  text-align: left;
+  margin-top: 5px;
+  font-weight: bolder;
+  color: #3e3e3e;
+}
+
+.organize {
+  font-family: cocogoose, sans-serif;
+  font-style: italic;
+  font-size: 9px;
+  font-weight: bold;
+  text-align: left;
+}
+
+.speaker-description {
+  font-family: cocogoose, sans-serif;
+  word-break: keep-all;
+  text-align: left;
+  font-size: 9px;
+  line-height: 14px;
+}
+
+.grid-3 {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+}
+
+.partner-bg {
+  background-repeat: repeat;
+  background-color: rgb(62, 65, 67);
+}
+
+.text-color {
+  color: #e1e1e1;
+}
+
+.w-max-2 {
+  max-width: 52px;
+}
+
+.toggle-1 {
+  display: none !important;
+}
+
+.toggle-2 {
+  display: block;
+}
+
+.toggle-3 {
+  display: none !important;
+}
+
+#more {
+  width: 100%;
+}
+
+.about-text-lg-1 {
+  font-size: 28px;
+  font-weight: 900;
+  text-transform: uppercase;
+  width: 344px;
+}
+
+.about-text-lg-2 {
+  font-size: 28px;
+  font-weight: 500;
+  text-transform: uppercase;
+}
+
+.grid-4 {
+  display: grid;
+  grid-template-columns: 100%;
+  gap: 1rem;
+}
+
+.partners {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+  gap: 14px;
+}
+
+.bg-footer,
+.bg-footer-about {
+  background-color: #fff;
+}
```

Comment:
- [ ] Great job with your design. Please make sure you increase the `font-size` of this section of your `About page`  to make it as close as possible to the Figma design🙏 

Original Design | Your About Page
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/190468288-455e1c55-daae-472a-9702-489d549a1e58.png)  |  ![image](https://user-images.githubusercontent.com/74814780/190468344-2a2fd3b5-8aa6-4630-8c89-6fa6264643ed.png)